### PR TITLE
Guiding principles for BLISS

### PR DESCRIPTION
**Pull request by SylLutze**
Would love thoughts on language and/or ideas! 

Also - I think this might create some merge conflicts with the references. I couldn't get them to work in the text with spaces in them so had to edit.

### CODE REVIEW

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 1]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+might be less likely to interact with healthcare, see themselves in research, and lose 
+opportunities as a result. In an effort to work toward institution-wide DEI goals and 
+advance our team’s research and publications, we created this guide to explore the 
+concepts of sex and gender and offer inclusive language alternatives.  
+
+One example of how proxy measures can affect the validity of research is in [Bauer_2022]_, 
+where researchers used 'women' as a proxy for people with a uterus. The researchers were 
+trying to find the incidence of uterine cancer, but by using 'women' as the denominator, 
+even though not all women have a uterus, cancer rates were thought to be lower than was true 
+in real lif. This was especially prominent in Black women who are more likely to have had
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:71]:
```suggestion
in real life. This was especially prominent in Black women who are more likely to have had 
```

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 2]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+even though not all women have a uterus, cancer rates were thought to be lower than was true 
+in real lif. This was especially prominent in Black women who are more likely to have had 
+a hysterectomy. This example highlights the importance of using a measure that is actually 
+tied to the research question.  
+
+A second example of the potential impact on research validity is equating differences in 
+results to only sex or gender. As explained in [Ritz_and_Greaves_2022]_, both sex and gender 
+can lead to biological changes and untangling these pathways is complex or impossible. Often, 
+researchers will report findings as "sex differences" implying that they are a result of sex, 
+when the possibility that gendered behavior or norms was the cause was never explored.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:79]:
```suggestion
researchers will report findings as "sex differences," implying that they are a result of sex, 
without exploring the role of gendered behavior or norms.
```

Nothing wrong here, just found this sentence a bit hard to follow!

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 3]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+information that users can understand the spirit of the guide and apply that to their unique needs.  
+
+**3. This is work**
+
+We acknowledge that there is a cost associated with a transition to gender inclusive language 
+practices from standard, gendered language. Although using gender inclusive language is work, 
+we highlight here that there are many benefits to incorporating more inclusive language into 
+everyday public health research practices.  
+
+Gender inclusive language can often by longer or more verbose than gendered language, but we
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:103]:
```suggestion
Gender inclusive language can often be longer or more verbose than gendered language, but we 
```

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 4]
```diff
@@ -100,8 +167,8 @@ DR paper; IV Iron communications and IDM Symposium)
 
 5.0 References
 ++++++++++++++
-.. [Bauer, 2022]
+.. [Bauer_2022]
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:170]: Yep, it was the spaces that was making these not work 👍 And now that you've actually referenced them above, Sphinx won't complain about them being here without being cited.

**Review by zmbc — APPROVED** [review]
Awesome work!

**Review by alibow — APPROVED** [review]

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 5]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+Inclusive language use is important, and we want to begin by acknowledging the 
+potential harm that using gendered language can create. ‘Sex’ and ‘gender’ are 
+not synonymous, unidimensional traits. [Bauer_2022]_ acknowledges this and explains 
+how common sex and gender proxies such as chromosomes or sex organs are imperfect 
+and not fully inclusive of lived experiences.  
+
+Using imperfect proxies can create real harm by alienating individuals, and by 
+creating invalid research. People who are not cisgendered (a diverse group including 
+intersex, nonbinary, transgender and other gender nonconforming individuals) experience 
+gendered language frequently which can be alienating and cause harm. These individuals
```
Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/index.rst:61]: I think this last sentence (starting with 'People who are not cis...') could actually be removed, it feels a little redundant to me.

Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/index.rst:61]:
```suggestion
People who are intersex, transgender, nonbinary, and/or have other gender-nonconforming identities
```

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 6]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+
+**1. This work is important**
+
+Inclusive language use is important, and we want to begin by acknowledging the 
+potential harm that using gendered language can create. ‘Sex’ and ‘gender’ are 
+not synonymous, unidimensional traits. [Bauer_2022]_ acknowledges this and explains 
+how common sex and gender proxies such as chromosomes or sex organs are imperfect 
+and not fully inclusive of lived experiences.  
+
+Using imperfect proxies can create real harm by alienating individuals, and by
```
Comment (pletale) [inline docs/source/model_design/general_reference_material/bliss/index.rst:58]:
```suggestion
Using imperfect proxies and gendered language can create real harm by alienating individuals, and by 
```

**Review by pletale — APPROVED** [review]
Looks great - thank you Syl!

File: docs/source/model_design/general_reference_material/bliss/index.rst (hunk 6 above)
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:58]:
```suggestion
Using imperfect proxies can create harm by alienating individuals, and by 
```
no need to risk it sounding like we are embellishing or undercuting this

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 7]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+
+Lastly, we hope to inspire change in other researchers by presenting our studies with gender 
+inclusive language and model practices we believe in. For all of these reasons, we feel this 
+guide is very important.  
+
+**2. This work is evolving**
+
+Language is constantly evolving, so there is rarely a single “right” choice and the accepted 
+choice may evolve over time. Therefore, this guide will be a living document that we will 
+regularly review and update. We will hold ourselves accountable for aligning the guide with
```
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:89]: Do we have a date somewhere to note when we have last updated the doc? I guess it is all logged in git, but it might be good to include it in the readthedocs version explicitly.

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 8]
```diff
@@ -44,7 +44,74 @@ Add an introductory paragraph here (include visual of 'What this is vs. what thi
...
+guide is very important.  
+
+**2. This work is evolving**
+
+Language is constantly evolving, so there is rarely a single “right” choice and the accepted 
+choice may evolve over time. Therefore, this guide will be a living document that we will 
+regularly review and update. We will hold ourselves accountable for aligning the guide with 
+the current preferred language, terminology and approaches.  
+
+While we hope this can be up to date and comprehensive, no guide can anticipate every use case,
```
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:92]:
```suggestion
While we hope this can be up-to-date and comprehensive, no guide can anticipate every use case, 
```

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/general_reference_material/bliss/index.rst (hunk 5 above)
Comment (SylLutze) [reply docs/source/model_design/general_reference_material/bliss/index.rst:61]: Fair! I combined with with the sentence below and shortened them to just include the examples rather than statement and then separately examples.
